front/lib/api/user.ts

```diff
@@ -140,41 +140,39 @@ export async function fetchRevokedWorkspace(
 }
 
 export async function getUserWithWorkspaces(
-  user: UserResource
+  user: UserResource,
+  populateExtensionConfig = false
 ): Promise<UserTypeWithWorkspaces> {
   const { memberships } = await MembershipResource.getActiveMemberships({
     users: [user],
   });
+  const workspaceIds = memberships.map((m) => m.workspaceId);
   const workspaces = await Workspace.findAll({
     where: {
-      id: memberships.map((m) => m.workspaceId),
+      id: workspaceIds,
     },
   });
 
+  const configs = populateExtensionConfig
+    ? await ExtensionConfigurationResource.internalFetchForWorkspaces(
+        workspaceIds
+      )
+    : [];
```

Comment:
I think it would be cleaner to keep this in /me endpoint and not change this or auth_wrapper

Reply:
Yes I started with that but that would mean that I would have to loop on all workspaces from my `user` which is a `UserTypeWithWorkspace` to fetch the potential config and edit the subtype? That looked even more fishy to me. 

This wrapper is only used for /me which is the only route without a workspace on the url. 

I _think_ the cleanest way would be to sort out why we have both a WorkspaceType and a LightWorkspaceType, we could harmonize and keep just one with those same optional attributes. I'm pinging the team to discuss it on Slack.